diff --git a/pkgs/development/interpreters/perl/5.16/default.nix b/pkgs/development/interpreters/perl/5.16/default.nix
index a8c1754..eff636c 100644
--- a/pkgs/development/interpreters/perl/5.16/default.nix
+++ b/pkgs/development/interpreters/perl/5.16/default.nix
@@ -71,6 +71,8 @@ stdenv.mkDerivation rec {
     lib/Net/hostent.t \
     dist/IO/t/{io_multihomed.t,io_sock.t} \
     t/porting/{maintainers.t,regen.t} \
+    dist/Net-Ping/t/*.t \
+    cpan/autodie/t/truncate.t \
     cpan/Socket/t/get{name,addr}info.t \
   '' + optionalString stdenv.isFreeBSD ''
     cpan/CPANPLUS/t/04_CPANPLUS-Module.t \
